client/src/pages/learn.tsx

```diff
@@ -105,6 +114,13 @@ function LearnPage({
               onDonationAlertClick={onDonationAlertClick}
               isDonating={isDonating}
             />
+            {isSignedIn && (
+              <ProgressIndicator
+                completedChallengeCount={completedChallengeCount}
+                username={username}
+                pathname={path}
+              />
+            )}
```

Comment:
```suggestion
            )}
            <h2 className='sr-only'>{t('settings.headings.certs')}</h2>
```

Now that we have a heading for the progress summary, I think it makes sense to have one for the certifications. We can visually hide it so it doesn't ruin the aesthetics of the page.

Reply:
Done. 🙌🏽